spec/services/sufia/workflow/changes_required_notification_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+RSpec.describe Sufia::Workflow::ChangesRequiredNotification do
+  let(:depositor) { create(:user) }
+  let(:cc_user) { create(:user) }
+  let(:work) { create(:generic_work, user: depositor) }
+  let(:entity) { create(:sipity_entity, proxy_for_global_id: work.to_global_id.to_s) }
+  let(:comment) { double("comment", comment: 'A pleasant read') }
+  let(:recipients) { { 'to' => [depositor], 'cc' => [cc_user] } }
+
+  describe ".send_notification" do
+    it 'sends a message to all users' do
+      expect(depositor).to receive(:send_message).once.and_call_original
```

Comment:
I would expect the approving user instance to receive the `:send_message` call as the user who initiates the state change.